src/constants/url.ts

```diff
@@ -27,7 +27,7 @@ export const DEFAULT_OAUTH_LOGOUT_URL = 'https://oauth.deriv.com/oauth2/sessions
 export const DEFAULT_OAUTH_ORIGIN_URL = 'https://oauth.deriv.com';
 
 const SocketURL = {
-    [URLConstants.derivP2pProduction]: 'blue.derivws.com',
+    [URLConstants.derivP2pProduction]: 'green.derivws.com',
```

Comment:
Why are we changing to green? Isn't blue used for production?

Reply:
@farrah-deriv we should be using green right?

Reply:
I believe we should be using green @rupato-deriv 

<img width="1066" alt="Screenshot 2024-09-26 at 2 22 17 PM" src="https://github.com/user-attachments/assets/e0b2f75f-ecff-45be-8305-11b30433f67a">